magit-file-icons.el

```diff
@@ -55,12 +55,52 @@
   :type 'boolean
   :group 'magit-file-icons)
 
+(defcustom magit-file-icons-icon-backend 'nerd-icons
+  "Icon backend for magit-file-icons. Set this to nil
+if you want to customize `magit-file-icons-icon-for-file-func'
+or `magit-file-icons-icon-for-dir-func'."
+  :type 'symbol
+  :group 'magit-file-icons)
+
+(if (not (require 'nerd-icons nil t)) (funcall
```

Comment:
I don't think this check is necessary if we have `nerd-icons` in the `Package-Requires` clause